conf/local.conf.js

```diff
@@ -0,0 +1,23 @@
+const master = require('../wdio.conf')
```

Comment:
seems good, however I would prefer to move this config into `test/e2e/config`, because conf at the root folder could let developers think it is related to the whole app, what do you think ?